sdk/communication/communication-short-codes/test/public/utils/testProgramBriefAttachment.ts

```diff
@@ -2,10 +2,18 @@
 // Licensed under the MIT license.
 
 import { ProgramBriefAttachment, ShortCodesClient } from "../../../src";
+import { isPlaybackMode } from "@azure-tools/test-recorder";
+import { v1 as uuid } from "uuid";
 
 export function getTestProgramBriefAttachment(): ProgramBriefAttachment {
+  let attachmentId = uuid();
+
+  if (isPlaybackMode()) {
+    attachmentId = "9d787bd6-07fc-4c7b-8e57-17f1fee41298";
+  }
```

Comment:
Instead of reassigning the value if is playback mode, I would suggest refactoring, so it is a constant value that is only assigned once.
```suggestion
  const attachmentId = isPlaybackMode() ? "9d787bd6-07fc-4c7b-8e57-17f1fee41298" : uuid();
```

Reply:
So, the thing is that tests run in parallel and they would target the same resource, so they would eventually conflict (that's actually the core problem here).